app/decorators/manageiq/providers/autosde/storage_manager/storage_manager_decorator.rb

```diff
@@ -0,0 +1,47 @@
+Module ManageIQ::Providers
+  class AutoSDE::StorageManagerDecorator < MiqDecorator
+    def self.fonticon
+      'pficon pficon-server'
+    end
+
+    def fonticon
+      nil
+    end
+
+    def fileicon
+      "svg/vendor-#{image_name}.svg"
+    end
+
+    def quadicon
+      icon = {
+        :top_left     => {},
+        :top_right    => {},
+        :bottom_left  => {
+          :fileicon => fileicon,
+          :tooltip  => ui_lookup(:model => type)
+        },
+        :bottom_right => QuadiconHelper.provider_status(authentication_status, enabled?)
+      }
+
+      # Due to the lack of the separate STI classes this has to be done :(
+      if supports_block_storage?
```

Comment:
Some AutoSDE managers don't support block storage? Or why this condition is necessary?

Reply:
currently only block storage. removed it.